pontoon/projects/utils.py

```diff
@@ -0,0 +1,5 @@
+def filter_users_by_project_visibility(project, users):
```

Comment:
Can we instead move this to `pontoon.base.models` and use `User.add_to_class`?
https://stackoverflow.com/questions/1642779/django-custom-managers-for-user-model

Reply:
Done :-)